com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -24,8 +24,20 @@ public sealed class NetworkObject : MonoBehaviour
         internal uint GlobalObjectIdHash;
 
 #if UNITY_EDITOR
+        // HEAD: TEST ONLY TEMP IMPL
```

Comment:
I like that there is a comment here, but I think we could be a bit more descriptive, like "This variable is only to enable testing, do not reference in code.  Will be removed with a future improved implementation"